Resources/Locale/en-US/accent/german.ftl

```diff
@@ -0,0 +1,221 @@
+
+accent-german-words-8 = steak
+accent-german-words-replace-8 = schnitzel
+
+accent-german-words-9 = steaks
+accent-german-words-replace-9 = schnitzels
+
+accent-german-words-10 = beer
+accent-german-words-10-2 = beers
+accent-german-words-replace-10 = bier
+
+accent-german-words-11 = that was an order
+accent-german-words-replace-11 = das war ein Befehl
+
+accent-german-words-12 = butterfly
+accent-german-words-replace-12 = schmetterling
+
+accent-german-words-13 = my god
+accent-german-words-replace-13 = mein gott
+
+accent-german-words-14 = my
+accent-german-words-replace-14 = mein
+
+accent-german-words-15 = here
+accent-german-words-replace-15 = hier
+
+accent-german-words-16 = idiot
+accent-german-words-replace-16 = dummkopf
+
+accent-german-words-17 = idiots
+accent-german-words-replace-17 = dummköpfe
+
+accent-german-words-18 = sec
+accent-german-words-replace-18 = sek
+
+accent-german-words-19 = machine
+accent-german-words-replace-19 = maschine
+
+accent-german-words-20 = machines
+accent-german-words-replace-20 = maschinen
+
+accent-german-words-21 = watch out
+accent-german-words-replace-21 = achtung
+
+accent-german-words-22 = music
+accent-german-words-replace-22 = musik
+
+accent-german-words-23 = captain
+accent-german-words-replace-23 = kapitän
+
+accent-german-words-24 = kebab
+accent-german-words-replace-24 = döner
+
+accent-german-words-25 = mouse
+accent-german-words-replace-25 = maus
+
+accent-german-words-26 = what
+accent-german-words-replace-26 = wat
+
+accent-german-words-27 = thank you
+accent-german-words-replace-27 = dankeschön
+
+accent-german-words-28 = thanks
+accent-german-words-replace-28 = danke
+
+accent-german-words-29 = bless you
+accent-german-words-replace-29 = gesundheit
+
+accent-german-words-30 = flamethrower
+accent-german-words-replace-30 = flammenwerfer
+
+accent-german-words-31 = ghost
+accent-german-words-replace-31 = poltergeist
+
+accent-german-words-32 = weed
+accent-german-words-replace-32 = kraut
+
+accent-german-words-33 = vodka
+accent-german-words-replace-33 = wodka
+
+accent-german-words-34 = backpack
+accent-german-words-replace-34 = rucksack
+
+accent-german-words-35 = medicine
+accent-german-words-replace-35 = medizin
+
+accent-german-words-36 = doc
+accent-german-words-replace-36 = dok
+
+accent-german-words-37 = doctor
+accent-german-words-replace-37 = doktor
+
+accent-german-words-38 = doctors
+accent-german-words-replace-38 = doktors
+
+accent-german-words-39 = stupid
+accent-german-words-replace-39 = dumm
+
+accent-german-words-40 = car
+accent-german-words-replace-40 = auto
+
+accent-german-words-41 = wonderful
+accent-german-words-replace-41 = wunderbar
+
+accent-german-words-42 = warning
+accent-german-words-replace-42 = warnung
+
+accent-german-words-43 = warnings
+accent-german-words-replace-43 = warnungen
+
+accent-german-words-44 = and
+accent-german-words-replace-44 = und
+
+accent-german-words-45 = officer
+accent-german-words-replace-45 = offizier
```

Comment:
remove this translation

Reply:
Does this also apply to "commander"?
Since that's also just the German form of the word and I can't tell if this is because this replacement in particular is inaccurate somehow or if I'm supposed to remove all military ranks.